src/config.docker.py

```diff
@@ -6,3 +6,4 @@
 JOB_EXPIRATION_MINUTES = int(
     os.environ.get("JOB_EXPIRATION_MINUTES", 0)
 )  # infinite by default
+PLUGINS = list(filter(None, os.environ.get("MQUERY_PLUGINS", "").split()))
```

Comment:
maybe `,` is better separator than whitespace? 🤔 But then it needs strip() for `", "`